doc/07-Operations.md

```diff
@@ -0,0 +1,128 @@
+# Operations
+
+Once Icinga DB is installed and configured, it usually runs silently in the background.
+This section is a loose collection of various topics to keep it that way.
+
+## Monitor Icinga DB
+
+It is strongly recommended to monitor the monitoring.
+
+There is a built-in [`icingadb` check command](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#icingadb) in the Icinga 2 ITL.
+It covers several potential errors, including operations that take too long or invalid high availability scenarios.
+Even if the Icinga DB has crashed, checks will still run and Icinga 2 would generate notifications.
+
+In addition, both the Redis® and the relational database should be monitored.
+There are predefined check commands in the ITL to choose from.
+
+- [`redis`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#redis)
+- [`mysql`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#mysql)
+- [`mysql_health`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#mysql_health)
+- [`postgres`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#postgres)
+
+A simpler approach would be to check if the processes are running, e.g.,
+with [`proc`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#procs) or [`systemd`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#systemd).
+
+## Backups
+
+There are only two things to back up in Icinga DB.
+
+1. The configuration file in `/etc/icingadb` and
+2. the relational database, using `mysqldump`, `mariadb-dump` or `pg_dump`.
+
+!!! warning
+
+    When creating a database dump for MySQL or MariaDB with `mysqldump` or `mariadb-dump`,
+    use the [`--single-transaction` command line argument flag](https://dev.mysql.com/doc/refman/8.4/en/mysqldump.html#option_mysqldump_single-transaction)
+    to not lock the whole database while the backup is running.
+
+## Automatic Restart
+
+<!-- NOTE: Would be obsolete after https://git.icinga.com/packages/icingadb/-/merge_requests/10 -->
```

Comment:
Looks liks the MR is already approved. I'll add a remincer to it to update this :)

Reply:
Thanks for cross-checking. However, I am still not quite sure if we should proceed with this MR, making the behavior described in this section the default.

Reply:
Thanks for commenting on the MR over at the GitLab. I would resolve this conversation here :)